src/modules/hg_branch.rs

```diff
@@ -76,6 +76,11 @@ fn get_hg_branch_name(ctx: &Context) -> String {
         .unwrap_or_else(|_| "default".to_string())
 }
 
+fn get_hg_topic_name(ctx: &Context) -> String {
+    std::fs::read_to_string(ctx.current_dir.join(".hg").join("topic"))
+        .map(|s| s.trim().into())
+        .ok()
+
```

Comment:
I think most of these issues are caused by a missing `}` here. I think this should be fix it
```suggestion
}

```